### PR TITLE
Summary Chapter in 3.1 Re-Order #488

### PR DESCRIPTION
**Pull request by rcorsi**
This PR covers issue #488 

- [x] This PR handles the issue and requires no additional PRs.
- [x] You have validated the need for this change.

**What did this PR accomplish?**

Re-ordered a few of the Summary items to match the order that they are presented in the chapter.

### CODE REVIEW

File: document/3-The_OWASP_Testing_Framework/1-Penetration_Testing_Methodologies.md [hunk 1]
```diff
@@ -3,10 +3,10 @@
...
 
 - OWASP Testing Guide
-- PCI Penetration Testing Guide
 - Penetration Testing Execution Standard
-- NIST 800-115
+- PCI Penetration Testing Guide
 - Penetration Testing Framework
+- NIST 800-115
 - Information Systems Security Assessment Framework (ISSAF)
 - Open Source Security Testing Methodology Manual (OSSTMM)
```
Comment (victoriadrake) [inline document/3-The_OWASP_Testing_Framework/1-Penetration_Testing_Methodologies.md:11]:
```suggestion
- Technical Guide to Information Security Testing and Assessment (NIST800-115)
  - Information Systems Security Assessment Framework (ISSAF)
  - Open Source Security Testing Methodology Manual (OSSTMM)
```

**Comment by victoriadrake** [comment]
Thank you @rcorsi! Just one small suggestion from me.

**Comment by ThunderSon** [comment]
@victoriadrake Just did it. Haha felt like just pushing it.

**Comment by rcorsi** [comment]
Glad you made the change @victoriadrake.  I was wondering why the full title was not already there.
Just wondering about the ISSAF and OSSTMM indenting.  Are those two really within NIST 800-115?
I'm not familar with NIST 800-115, and I do see OSSTMM mentioned in that document, but it doesn't seem to be an independent methodology apart from NIST one.

**Comment by rcorsi** [comment]
Also the OWASP is not expanded at all, apart from being in the summary list.  Should an entry be added for that one too?   Making mention to "This document" and can also mention the "Mobile" version that is already in the reference section.  Not sure how to word it exactly.

Is it worth adding that bit of info?

**Comment by ThunderSon** [comment]
No they're not. I'll update this tomorrow :) That commit was a bit tricky, no worries though.

**Pull request merged**